input/docs/cs/imagenet_classes.md

```diff
@@ -0,0 +1,1014 @@
+title: Zobrazení 1000 tříd databáze ImageNet pomocí konvoluční neuronové sítě Alexnet
+Description: Jednoduché zobrazení 1000 klasifikačních tříd z obrázkové databáze ImageNet v proměnné pro snadné vyhledávání jednotlivých tříd v abecedně seřazeném seznamu.
+---
+```matlab
+net = alexnet; % načíst sít
+ClassNames = net.Layers(end).ClassNames; %uložit všechny názvy tříd do proměnné
+ClassNamesSorted = sort(ClassNames); % třídit názvy sítí podle abecedy
```

Comment:
tak teď mi došlo, že to je trochu nesmysl řadit to abecedně. Pak by nepasovaly ty třídy s tím co je v alexnet ,atd.. Tak jenom vyhoď ten sortící řádek, dej pryč `ClassNamesSorted` a uprav ten výstup.